tests/test_mstump.py

```diff
@@ -50,7 +55,7 @@ def test_get_first_mstump_profile(T, m):
     left_I = left_I[start, :]
 
     M_T, Σ_T = core.compute_mean_std(T, m)
-    right_P, right_I = _get_first_mstump_profile(start, T, m, excl_zone, M_T, Σ_T)
+    right_P, right_I = _get_first_mstump_profile(start, T, T, m, excl_zone, M_T, Σ_T)
```

Comment:
This seems weird. I feel like `_get_first_mstump_profile` should detect whether one or two time series are being passed to it

Reply:
'stump._get_first_stump_profile` doesn't, so I kept it this way in `mstump`. The detection is done in the main `stump` function.

Reply:
Got it! I overlooked the fact that:

1. This is a private function
2. This sets us up in case we decide to allow AB-joins in the future (currently, this is blocked/omitted in the public function)